poliastro/twobody/classical.py

```diff
@@ -127,6 +127,10 @@ def argp(self):
     def nu(self):
         return self._nu
 
+    @property
```

Comment:
See comment below, this wouldn't be necessary. It's an internal method for IPython use: http://ipython.readthedocs.org/en/3.x/config/integrating.html#rich-display